src/main/java/step5/domain/RacingCar.java

```diff
@@ -0,0 +1,25 @@
+package step5.domain;
+
+public class RacingCar {
+
+    private String carName;
+    private int position;
+
+    public RacingCar(String carName) {
+        if (carName.length() > 5) throw new IllegalArgumentException();
```

Comment:
```suggestion
        if (carName.length() > 5) { 
            throw new IllegalArgumentException();
        }
```
저는 한줄짜리 if문이라도 중괄호를 넣는편입니다. 중괄호가 있어야 이 후 수정시에 실수를 줄여주는 것 같아요. 🙂